distribution/ltp-upstream/lite/runtest.sh

```diff
@@ -1,19 +1,7 @@
 #!/bin/bash
 
-# Copyright (c) 2011 Red Hat, Inc. All rights reserved. This copyrighted material 
-# is made available to anyone wishing to use, modify, copy, or
-# redistribute it subject to the terms and conditions of the GNU General
-# Public License v.2.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT ANY
-# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-# PARTICULAR PURPOSE. See the GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
-#
-# Author: Zhouping Liu <zliu@redhat.com> 
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Red Hat, Inc.
```

Comment:
Is the author removed on purpose?

Reply:
Yes, the author has left RedHat for more than 5 years. This test changes so much and no longer as it used to be, to avoid people send to the no-exist email account. So I remove it.